presentation/src/main/java/com/anytypeio/anytype/presentation/extension/MapExtension.kt

```diff
@@ -32,10 +32,15 @@ fun ObjectWrapper.Basic.getProperObjectName(): String? {
             snippet?.replace("\n", " ")?.take(MAX_SNIPPET_SIZE)
         }
         in SupportedLayouts.fileLayouts -> {
+            val fileName = if (name.isNullOrBlank()) "Untitled" else name.orEmpty()
```

Comment:
"Untitled" should be localized - consider getting string ressource in your GUI component.